lib/backend/postgres/index.ts

```diff
@@ -478,8 +489,10 @@ export class PostgresBackend implements Database {
 	 * executed once by a single instance.
 	 */
 	private async runDbMigrations(context: Context) {
```

Comment:
Maybe renaming `runDbMigrations`( which sounds redundant ) to `runCoreMigrations` would make more evident that these are the "autumndb" migrations

Reply:
Good point, the `Db` bit is obvious and redundant.

Reply:
Done, updated to `runCoreMigrations`